cves/CVE-2016-3081.yml

```diff
@@ -121,20 +130,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
+  answer: |
+    I could not find how the vulnerability was discovered. I looked at all the link in
+    the CVE and only found information about what the vulnerability is.
+  date: 2016-04-22
   automated:
-  apache:
-  contest:
+  apache: false
+  contest: true
```

Comment:
Might want to double check that this was found in a contest

Reply:
You're right. I misinterpreted the instructions as determine if the vulnerability was discovered by an Apache employee, or else it is in a contest. Will make that change.